docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -0,0 +1,55 @@
+### Contributors:
+* Yasuo Kida (independent)
+* Taro Yamamoto (Adobe)
+* Nat McCully (Adobe)
+* Kobayashi Toshi (independent)
+* KOBAYASHI Tatsuo (independent)
+* Atsushi Shimono (W3C)
+* Shinyu MURAKAMI (Vivliostyle)
+* MURATA Makoto (Keio University)
+
+***
+
+[CITPC(Council for the Promotion of Character Information Technology)](http://citpc.jp) is a general incorporated association in Japan.  CITPC aims to enhance the interoperability of character information processing in Japan.  Most of the font vendors in Japan are members of CITPC.  
+
+UAX 50 was designed as a cornerstone of vertical writing for the Web and EPUB.  CITPC is grateful to the Unicode Consortium for the timely development of UAX 50 and its ongoing maintenance.   

```

Comment:
```suggestion
[UAX 50](https://www.unicode.org/reports/tr50/) was designed as a cornerstone of vertical writing for the Web and EPUB.  CITPC is grateful to the Unicode Consortium for the timely development of UAX 50 and its ongoing maintenance.   

```

We can add a link to the current version of UAX 50.

Reply:
as a letter to Unicode, I believe we don't need a link, but I really agree that adding a link is helpful for reader on this repository.